dottydoc/test/BaseTest.scala

```diff
@@ -12,7 +12,7 @@ import model.Package
 trait DottyTest {
   dotty.tools.dotc.parsing.Scanners // initialize keywords
 
-  implicit var ctx: FreshContext = {
+  implicit val ctx: FreshContext = {
```

Comment:
This change seems irrelevant to the PR/commit. It shouldn't appear in the commit.